### PR TITLE
Prefer Jackson over JSON-B in the documentation

### PR DESCRIPTION
**Pull request by gsmet**

### CODE REVIEW

**Comment by geoand** [comment]
I'm definitely +1 for this.

Did we have a mailing list discussion on this?

**Comment by gsmet** [comment]
No, we just had a Zulip discussion, we still keep JSON-B around so I don't think that's such a big deal. I can post on the list though.

**Review by geoand — APPROVED** [review]

**Review by gastaldi — APPROVED** [review]

**Comment by gsmet** [comment]
Let's wait until Monday and see if someone complains about it.

File: docs/src/main/asciidoc/rest-json.adoc [hunk 1]
```diff
@@ -181,58 +184,58 @@ It will allow to narrow down the number of JAX-RS providers (which can be seen a
 
 === Configuring JSON support
 
-==== JSON-B
+==== Jackson
 
-Quarkus makes it very easy to configure various JSON-B settings via CDI beans. The simplest (and suggested) approach is to define a CDI bean of type `io.quarkus.jsonb.JsonbConfigCustomizer`
-inside of which any JSON-B configuration can be applied.
+As stated above, Quarkus provides the option of using https://github.com/FasterXML/jackson[Jackson] instead of JSON-B via the use of the `quarkus-resteasy-jackson` extension.
```
Comment (Ladicek) [inline docs/src/main/asciidoc/rest-json.adoc:189]: Nitpicking: "Jackson instead of JSON-B" still reads a little like JSON-B is preferred.

Comment (gsmet) [reply docs/src/main/asciidoc/rest-json.adoc]: Ah ah, good catch. I just moved the sections, didn't think they would need changes.

I fixed it.

**Pull request merged**